README.md

```diff
@@ -307,20 +313,186 @@ IRIDA Next will attempt to autopopulate the `organism` column in the samplesheet
 
 # 3. Parameters
 
+## 3.1. Overview
+
 Parameters within a pipeline are defined in the `nextflow.config` file (under the `params` scope), and then the `nextflow_schema.json` file is updated to contain the parameters by running `nf-core schema build`. This is described in the [nf-core contributing to pipelines][nf-core-contributing-to-pipelines] documentation.
 
-As an example, the following is a subset of parameters from the [iridanext-example-nf][] pipeline.
+As an example, the following are excerpts from the [speciesabundance][] pipeline.
 
-```
+**nextflow.config**
+```groovy
 params {
+    // Input options
     input                      = null
-    genome                     = 'hg38'
-    igenomes_base              = 's3://ngi-igenomes/igenomes'
-    igenomes_ignore            = false
+
+    //Pipeline parameters
```

Comment:
Add space to match above "// Pipeline parameters"

Reply:
Done in [504bd95](https://github.com/phac-nml/pipeline-standards/pull/28/commits/504bd9518329359d2c79d238abb9381888c9841c)